_layouts/person.html

```diff
@@ -141,8 +141,8 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
   <section class="person-shows">
     {% for record in person.shows %}
     <div class="person-show">
-      <div class="person-show-poster">
-        <a href="{{ record.item.url }}" title="{{ record.item.title | escape_once }}">
+      <a href="{{ record.item.url }}" class="person-show-title" title="{{ record.item.title | escape_once }}">
```

Comment:
The class for this shouldn't be person-show-title... It has the poster inside it as well.